composer.json

```
@@ -15,7 +15,7 @@
 		"php": ">=5.4",
 		"ext-sockets": "*",
 		"ext-json": "*",
-		"ralouphie/getallheaders": "~2.0"
+		"ralouphie/getallheaders": "~2.0||^3.0.0"
```

```suggestion
		"ralouphie/getallheaders": "~2.0 || ~3.0"
```

For the sake of being picky let’s make them both of the same form.

---

Done.
I will squash if validated